CHANGES

```diff
@@ -9,6 +9,14 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Added functionality to create an EVP_PKEY from user data.  This
+     is effectively the same as creating an RSA, DH or DSA object and
+     assign them numbers and then assign them to an EVP_PKEY, but is
```

Comment:
Although I know what you mean - it is not very clear. Maybe somnething like...

This is equivalent to creating a EVP_PKEY using RSA, DH, or DSA methods.

Reply:
How about I simply remove "and assign them numbers"?